gruntfile.js

```diff
@@ -41,6 +44,25 @@ module.exports = function (grunt) {
                 files: './lib/**/*.js',
                 tasks: ['babel:dist']
             }
+        },
+        connect: {
+            server: {
```

Comment:
You actually don't need `connect` server here.
You can just use `file://` protocol to open local file.